feature-group-definitions/class-syntax.yml

```
@@ -1,4 +1,5 @@
 name: Classes
+description: The `class` JavaScript keyword defines a new class, which is a type of function that can be instantiated multiple times by using the `new` keyword.
```

I'm having a hard time with this one. The description isn't wrong, but I don't think it says much about what it does for developers. Trying something completely different here, just to get thinking about this:

```suggestion
description: Classes are an object-oriented syntax for JavaScript prototypes.
```

Maybe @Elchi3 has an idea here, as well.